### DISCUSSION TITLE
Issue with iCloud Calendar. Couldn't be visualized

### DISCUSSION THREAD

**Discussion by tunguskar** [totaldebug/atomic-calendar-revive#537] [Support] [answered]
Hi

I have setup the iCloud calendar and I can see it in HomeAssistant. Now I wanted to install this has component. 

One of my calendar entities is called: calendar.Family

The minimal config for revive: 

type: custom:atomic-calendar-revive
name: Calendar Card
enableModeChange: true
entities:
  - entitiy: calendar.family
    titleColor: red

What I get ist the following error message:

<img width="974" alt="image" src="https://user-images.githubusercontent.com/2837624/142003576-9f0a6d24-3093-4809-84ba-2769273a71d9.png">

Is this a known issue?

**Comment by marksie1988** [comment]
Please try change `calendar.family` to `calendar.Family` this will be case sensitive.

**Reply by tunguskar** [reply]
nope:
<img width="974" alt="image" src="https://user-images.githubusercontent.com/2837624/142021499-9258fb59-3bd6-485a-a03b-cefa51a4aad1.png">

The Entities look like this
<img width="1057" alt="image" src="https://user-images.githubusercontent.com/2837624/142021654-bde4fa67-bca1-42fd-83d4-105b23f175e1.png">

Left is the normal calendar with CalDAV and right is the revive:
![image](https://user-images.githubusercontent.com/2837624/142022264-671c7b71-91bd-4dea-aaab-00a0c1b14f7d.png)

Has this something to do with iCloud?

**Reply by tunguskar** [reply]
Is it possible to debug revive? Could it be, that it has no rights to get a reference to the calendar?

**Comment by tunguskar** [comment]
I also checked the *.js file:
<img width="293" alt="image" src="https://user-images.githubusercontent.com/2837624/142754053-c76b7f14-a0af-49b1-b3e5-0e99d038f143.png">

**Comment by marksie1988** [comment]
If you hit f12 you can see any errors in the console if there are any.

Otherwise this usually happens for two reasons, invalid config or invalid calendar.

**Reply by tunguskar** [reply]
@marksie1988  I get the following error message in the java script console: 
[Error] Failed to load resource: the server responded with a status of 400 (Bad Request) (undefined, line 0
<img width="698" alt="error" src="https://user-images.githubusercontent.com/2837624/142756155-32b07fc5-6e8d-40f6-bea7-e7f744f7497d.png">
)

**Reply by marksie1988** [reply, marked as answer]
Ok that means it can't find the calendar. 

Ah I just noticed your config has `entitiy` instead of `entity`

**Reply by tunguskar** [reply]
Thanks a lot @marksie1988 works now. Sorry for such a stupid mistake
